data/json/items/comestibles/dairy.json

```diff
@@ -242,6 +242,22 @@
     "fun": 7,
     "vitamins": [ [ "calcium", 16 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "cottage_cheese",
+    "name": { "str_sp": "cottage cheese" },
+    "copy-from": "cheese",
+    "weight": "34 g",
+    "spoils_in": "180 days",
+    "quench": 1,
+    "calories": 391,
+    "description": "Unaged curdled milk and whey, made by draining cheese rather than pressing like cheese curds. The retained whey keeps the curds loose and soft, giving a somewhat soupy texture to this lower calorie variety of cheese.",
```

Comment:
```suggestion
    "description": "Unaged curdled milk and whey, made by draining cheese rather than pressing like cheese curds.  The retained whey keeps the curds loose and soft, giving a somewhat soupy texture to this lower calorie variety of cheese.",
```